integration_tests/testsuite/test_util.py

```diff
@@ -54,10 +64,15 @@ def _generate_int64_token():
     return random.randint(-(2 ** 31), (2 ** 31)-1)
 
 
-def generate_logging_payload():
-    data = {'log_name': _generate_name(),
-            'token': _generate_hex_token()}
-    return data
+def generate_logging_payloads():
+    payloads = []
+    for l in list(Severity):
```

Comment:
If this is the only usage of the Severity enum, it might make sense to just treat those as a list of strings instead of an enum.